deploy/build-server/Dockerfile

```diff
@@ -0,0 +1,96 @@
+FROM ubuntu:20.04
+
+SHELL ["/bin/bash", "-c"]
+# To make it easier for build and release pipelines to run apt-get,
+# configure apt to not require confirmation (assume the -y argument by default)
+ENV DEBIAN_FRONTEND=noninteractive
+RUN echo "APT::Get::Assume-Yes \"true\";" > /etc/apt/apt.conf.d/90assumeyes
+
+# core dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    # azure devops agent
+    ca-certificates \
+    curl \
+    git \
+    jq \
+    iputils-ping \
+    libcurl4 \
+    libicu66 \
+    libssl1.0 \
+    libunwind8 \
+    netcat \
+    # ruby tests
+    freetds-bin \
+    # docker install
+    gnupg2 \
+    lsb-release \
+    # dotnet
+    apt-transport-https \
+    # brightbox
+    software-properties-common \
+    # dev build
+    zip \
+    && rm -rf /var/lib/apt/lists/*
+
+# install azure CLI
```

Comment:
It would be if you could explain why Azure CLI is required.  I would assume the build scripts from Azure Devops?